webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/ScrambleRequest.java

```diff
@@ -257,6 +268,9 @@ private static ByteArrayOutputStream createPdf(String globalTitle, Date creation
         // list of 333ni scrambles.
         // If we detect that we're dealing with 333mbf, then we will generate 1 sheet per attempt,
         // rather than 1 sheet per round (as we do with every other event).
+        
+        // for ordered scrambles, we recreate scrambleRequest so it contains only 1 scramble
+        // to fix this, we pass the attempt number
```

Comment:
I'm not sure what "pass the attempt number" is referring to here. Could you update this comment a bit?

nit: Also, if you look at the other comments in TNoodle, you'll see that we tend to write in full sentences. Could you do that as well for consistency?